charts/kusion/README.md.gotmpl

```diff
@@ -79,8 +85,8 @@ helm install kusion-release kusionstack/kusion \
```

Comment:
The addition of 'tr -d '
'' is a good practice to ensure no newlines are present in the base64 encoded string, which could cause issues when used in Helm values.